site/pages/api/duplicate-consequence.api.ts

```diff
@@ -43,7 +43,15 @@ const duplicateConsequence = async (req: NextApiRequest, res: NextApiResponse):
         }
 
         await upsertConsequence(
-            { ...consequenceToDuplicate, consequenceIndex: disruption.consequences.length },
+            {
+                ...consequenceToDuplicate,
+                consequenceIndex: disruption.consequences
+                    ? disruption.consequences.reduce(
+                          (max, c) => (c.consequenceIndex > max ? c.consequenceIndex : max),
+                          disruption.consequences[0].consequenceIndex,
```

Comment:
Could this be 0

Reply:
no because let's say i delete them all and i have a consequence with index of 1 i would want it to be 2